### PR TITLE
Fix some insecure links

### PR DESCRIPTION
**Pull request by ArmeF97**

### CODE REVIEW

File: Telegram/SourceFiles/ui/widgets/input_fields.cpp [hunk 1]
```diff
@@ -2361,7 +2361,7 @@ TextWithTags InputField::getTextWithAppliedMarkdown() const {
 	const auto &originalTags = _lastTextWithTags.tags;
 
 	// Ignore tags that partially intersect some http-links.
-	// This will allow sending http://test.com/__test__/test correctly.
```
Comment (john-preston) [inline Telegram/SourceFiles/ui/widgets/input_fields.cpp]: Why would you change that?..

Comment (ArmeF97) [reply Telegram/SourceFiles/ui/widgets/input_fields.cpp]: http is going to be shut down (sooner or later), so even a test link, it's better to upgrade it

File: Telegram/Resources/qrc/qmime/freedesktop.org.xml [hunk 2]
```diff
@@ -58,7 +58,7 @@
   <!ELEMENT sub-class-of EMPTY>
   <!ATTLIST sub-class-of type CDATA #REQUIRED>
 ]>
-<mime-info xmlns="http://www.freedesktop.org/standards/shared-mime-info">
```
Comment (stek29) [inline Telegram/Resources/qrc/qmime/freedesktop.org.xml]: [xmlns should not be changed](https://stackoverflow.com/questions/30707609/xml-namespace-uri-with-https)

File: Telegram/Resources/qrc/qmime/freedesktop.org.xml [hunk 3]
```diff
@@ -372,7 +372,7 @@
     <alias type="text/mathml"/>
     <sub-class-of type="application/xml"/>
     <glob pattern="*.mml"/>
-    <root-XML namespaceURI="http://www.w3.org/1998/Math/MathML" localName="math"/>
+    <root-XML namespaceURI="https://www.w3.org/1998/Math/MathML" localName="math"/>
```
Comment (stek29) [inline Telegram/Resources/qrc/qmime/freedesktop.org.xml]: Same thing applies here

**Comment by stek29** [comment]
You should not change links in xml, and the only change that's left is test.com one, which doesn't make much sense either, sorry.

**Pull request closed without merging**